superset/config.py

```diff
@@ -1066,6 +1066,12 @@ def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     return sql
 
 
+def NOTIFICATION_EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
+    header: Dict[str, Any], **kwargs: Any
```

Comment:
I believe this is message, right? Rather than header?

Reply:
yes, sorry. Will change but will wait for the rest of your review first